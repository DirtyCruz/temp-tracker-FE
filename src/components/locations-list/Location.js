import React from 'react';
import PropTypes from 'prop-types';

function Location({ location, temperature }) {
  return (
    <span>
      <h4>{location}</h4>
      <h5>{temperature}</h5>
    </span>
  );
}

Location.propTypes = {
  location: PropTypes.string.isRequired,
  temperature: PropTypes.number.isRequired
};

export default Location;
