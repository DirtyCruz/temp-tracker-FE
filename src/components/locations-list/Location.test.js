import React from 'react';
import { shallow } from 'enzyme';
import Location from './Location';

describe('Location component', () => {
  it('renders Location', () => {
    const wrapper = shallow(<Location location='location' temperature={6} />);
    expect(wrapper).toMatchSnapshot();
  });
});
