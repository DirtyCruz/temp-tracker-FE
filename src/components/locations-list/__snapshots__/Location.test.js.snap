// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Location component renders Location 1`] = `
<span>
  <h4>
    location
  </h4>
  <h5>
    6
  </h5>
</span>
`;
